.github/workflows/traclusdl.yml

```
@@ -0,0 +1,82 @@
+# and make sure unit tests run too with all supported rust versions
+
+name: CI
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+permissions: read
```

actionlint (1.7.12)

[error] "read" is invalid for permission for all the scopes. available values are "read-all", "write-all" or {}

(permissions)